src/js/libcs/List.js

```diff
@@ -209,9 +209,9 @@ List.consecutive = function(a) {
 };
 
 List.reverse = function(a) {
-    var erg = [];
-    for (var i = a.value.length - 1; i >= 0; i--) {
-        erg.push(a.value[i]);
+    var erg = new Array(a.value.length);
```

Comment:
I must say I'm surprised. Even accepting that the slice and reverse does touch each item of data twice instead of once, the time is still more than twice that of the native code, suggesting people would be better off reimplementing these functions instead of using the built-in versions. Strange! Too bad jsPerf isn't available.